.travis.yml

```diff
@@ -47,62 +36,145 @@ matrix:
     - os: linux
 
   include:
+    - os: linux
+      env: DIST=pangolin COMPILER=g++-4.7 CCOMPILER=gcc-4.7
+      addons:
+        apt:
+          sources: ['ubuntu-toolchain-r-test']
+          packages: ['g++-4.7']
+      compiler: gcc-4.7
 
-  # 1. Linux Clang Builds
+    - os: linux
+      env: DIST=pangolin COMPILER=g++-4.8 CCOMPILER=gcc-4.8
+      addons:
```

Comment:
Could you use the shorthand syntax here, the way it was before?
Eg. `addons: &gcc48` (this names a variable, which gets expanded if used as `*gcc48`). This will shorten the file and reduce the copy-paste-s.